File: app/src/main/java/com/eugene_poroshin/weatherforecast/fragments/OnOpenFragmentListener.java
package com.eugene_poroshin.weatherforecast.fragments;

public interface OnOpenFragmentListener {
    void onOpenForecastFragment();
    void onOpenForecastFragmentByCityName(String newCityName);
    void onOpenCityListFragment();
    void onOpenTempModePreferenceFragment();
}

File: app/src/main/java/com/eugene_poroshin/weatherforecast/fragments/OnOpenFragmentListener.kt
package com.eugene_poroshin.weatherforecast.fragments

interface OnOpenFragmentListener {
    fun onOpenForecastFragment()
    fun onOpenForecastFragmentByCityName(newCityName: String?)
    fun onOpenCityListFragment()
    fun onOpenTempModePreferenceFragment()
}